microsetta_admin/templates/email_stats_pulldown.html

```diff
@@ -0,0 +1,87 @@
+{% extends "sitebase.html" %}
+{% block head %}
+<link rel="stylesheet" type="text/css" href="/static/vendor/DataTables/DataTables-1.10.21/css/jquery.dataTables.css">
+<script src="/static/vendor/DataTables/DataTables-1.10.21/js/jquery.dataTables.js"></script>
+
+<script class="init">
+    function remove_error_messages () {
+        block = document.getElementById("search_errors");
+        if (block !== null) {
+            block.innerHTML = "";
+        }
+    };
+
+    $(document).ready(function() {
+        {% if resource is not none %}
+            $('#search_results').DataTable();
+        {% endif %}
+    });
+</script>
+{% endblock %}
+{% block content %}
+<h3>Account Summaries</h3>
+<div>
+    <form name="search_form" id="search_form" method="GET">
+        <table>
+            <tr>
+                <td><label for="email">Email: </label></td>
+                <td><input type="text" name="email" id="email" {% if info %} value="{{info.email}}"{% endif %}></td>
+                <td><label for="project1">Project: </label>
+                <input type="text" id="project1" name="project"><br/>
+                <td><input type="submit" value="Retrieve Account Summary"></td>
+            </tr>
+        </table>
+        {% if search_error %}
+            <p style="color:red" id=search_errors>
+            {% for error in search_error %}
+                {{error |e}}<br/>
+            {% endfor %}
+            </p>
+        {% endif %}
+        <script>
+           document.search_form.email.focus()
+        </script>
+    </form>
+    <br/>
+    Or upload a file and pull account summaries for all emails listed in the column "email"
+    <br/>
+    <form name="upload_csv" id="search_csv" method="POST" enctype="multipart/form-data" onsubmit="return remove_error_messages()">
+      <br/>
+      <label for="project2">Project: </label>
+      <input type="text" id="project2" name="project"><br/>
+      <input type=file name=file><br/>
+      <input type=submit value=Upload>
+    </form>
+
+    <hr>
+    {% if resource is not none %}
+    <div class="result_container">
+        <div>
+            <table id="search_results" class="display" style="width:100%">
+                <thead>
+                    <tr>
+                        {% for col in resource.columns %}
+                        <th>{{col |e}}</th>
+                        {% endfor %}
+                    </tr>
+                </thead>
+                <tbody>
+                {% for row in resource.index %}
+                    <tr>
+                        {% for col in resource.columns %}
+                        {% if col == "account_id" %}
+                        <td><a href="{{ ui_endpoint }}/accounts/{{resource.at[row, col] |e}}">{{resource.at[row, col] |e}}</a></td>
```

Comment:
Indexing each value is going to be pretty inefficient. Would it make sense to use the dataframe's to_html method? see https://stackoverflow.com/a/45701949 or https://stackoverflow.com/a/58112545

Reply:
That's pretty neat.  It's much simpler that way, though I'm not sure I can add a link to view the account this way.  I'll push up this version, let me know which you prefer.

Reply:
Here's one way we add in the account links (https://github.com/biocore/microsetta-admin/blob/master/microsetta_admin/templates/search_result.html#L52-L65), although it does differ some from the SO posts though with the benefit of avoiding a `.at`